e2e/images_test.go

```diff
@@ -0,0 +1,53 @@
+package e2e
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"gotest.tools/assert"
+	"gotest.tools/fs"
+	"gotest.tools/icmd"
+)
+
+var (
+	reg      = regexp.MustCompile("Digest is (.*).")
+	expected = `REPOSITORY             TAG    APP NAME
+%s        push-pull
+a-simple-app           latest simple
+b-simple-app           latest simple
+`
+)
+
+func TestImageList(t *testing.T) {
+	runWithDindSwarmAndRegistry(t, func(info dindSwarmAndRegistryInfo) {
```

Comment:
As `runWithDindSwarmAndRegistry` is defined in `e2e/pushpull_test.go` maybe we can move the method in `e2e/helper_test.go`. This way this test only depends on the helper and not on an other test file.

Reply:
Done